photon-client/src/views/PipelineViews/PnPTab.vue

```diff
@@ -60,7 +60,7 @@
         },
         data() {
             return {
-                targetList: ['2020 High Goal Outer', '2020 High Goal Inner', '2019 Dual Target', 'Power Cell (7in)'],
+                targetList: ['2020 High Goal Outer', '2020 High Goal Inner', '2019 Dual Target', 'Power Cell (7in)', '2016 High Goal'], //Keep in sync with TargetModel.java
```

Comment:
In general this really should be sent by the backend. We should make an issue to fix this but not in this PR.

Reply:
https://github.com/PhotonVision/photonvision/issues/182 in